src/test/javascript/spec/component/shared/metis/postings-footer/post-footer/post-footer.component.spec.ts

```diff
@@ -1,30 +1,61 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { PostFooterComponent } from 'app/shared/metis/posting-footer/post-footer/post-footer.component';
-import { ArtemisTranslatePipe } from 'app/shared/pipes/artemis-translate.pipe';
-import { MockComponent, MockModule, MockPipe } from 'ng-mocks';
+import { MockComponent, MockModule } from 'ng-mocks';
 import { FaIconComponent } from '@fortawesome/angular-fontawesome';
 import { PostReactionsBarComponent } from 'app/shared/metis/posting-reactions-bar/post-reactions-bar/post-reactions-bar.component';
 import { ArtemisCoursesRoutingModule } from 'app/overview/courses-routing.module';
 import { MetisService } from 'app/shared/metis/metis.service';
 import { MockMetisService } from '../../../../../helpers/mocks/service/mock-metis-service.service';
 import { getElements } from '../../../../../helpers/utils/general.utils';
-import { metisPostExerciseUser1, metisPostLectureUser1, metisPostLectureUser2, metisTags } from '../../../../../helpers/sample/metis-sample-data';
+import { PostService } from 'app/shared/metis/post.service';
+import { MockPostService } from '../../../../../helpers/mocks/service/mock-post.service';
+import { AnswerPostService } from 'app/shared/metis/answer-post.service';
+import { MockAnswerPostService } from '../../../../../helpers/mocks/service/mock-answer-post.service';
+import { PostComponent } from 'app/shared/metis/post/post.component';
+import { AnswerPostComponent } from 'app/shared/metis/answer-post/answer-post.component';
+import { AnswerPostCreateEditModalComponent } from 'app/shared/metis/posting-create-edit-modal/answer-post-create-edit-modal/answer-post-create-edit-modal.component';
+import { TranslatePipeMock } from '../../../../../helpers/mocks/service/mock-translate.service';
```

Comment:
Could we use absolute imports (starting with ``app/...``) rather than relative paths? I'd advise this in general :)

Reply:
Seems like you started the failing import with ``src/test/...``, while it actually needs to start with ``app/...``. See the other imports above in lines 11 and 12 for reference.

Reply:
Exactly, the problem happens for files in the test directory.

Reply:
If we start them with helpers it can't find the module. Lets take "TranslatePipeMock" for instance. It is imported the same way I have done in other files too. All files to be precise. See the screenshot below.
![image](https://user-images.githubusercontent.com/67344158/169141790-bb3bb6c1-3168-4564-b3d6-db80c5c80dd6.png)